src/Dflydev/FigCookies/SetCookies.php

```diff
@@ -86,23 +86,21 @@ public function renderIntoSetCookieHeader(ResponseInterface $response) : Respons
      * Create SetCookies from a collection of SetCookie header value strings.
      *
      * @param string[] $setCookieStrings
-     * @return static
      */
-    public static function fromSetCookieStrings(array $setCookieStrings) : self
+    public static function fromSetCookieStrings(array $setCookieStrings): self
     {
-        return new static(array_map(function (string $setCookieString) : SetCookie {
+        return new self(array_map(static function (string $setCookieString): SetCookie {
```

Comment:
class is not final, means this is a behavior change

Reply:
Reverted.